web/package.json

```diff
@@ -152,7 +155,8 @@
   "babel": {
     "plugins": [
       "react-hot-loader/babel",
-      "@babel/plugin-proposal-class-properties"
+      "@babel/plugin-proposal-class-properties",
+      "@babel/plugin-syntax-dynamic-import"
```

Comment:
This doohickey has to be turned on for webpack's dynamic import (née lazy-loading) to work, because shut up I guess

Reply:
😂